synapse/lib/lmdblayer.py

```diff
@@ -276,17 +277,19 @@ def migrateProvPre010(self, newslab):
             proputf8 = lkey[32:]
             valu, indx = s_msgpack.un(lval)
 
-            # <prop><00><indx>
```

Comment:
did we not have migration / regression tests for this function???

Reply:
nope

Reply:
:cry: